integration-tests/contracts/SimpleStorage.sol

```diff
@@ -5,9 +5,9 @@ contract ICrossDomainMessenger {
 }
 
 contract SimpleStorage {
+    bytes32 public value;
     address public msgSender;
     address public xDomainSender;
-    bytes32 public value;
```

Comment:
Why did you move this variable? Just curious

Reply:
First variable is the easiest key in the whole storage layout ;)